src/api/app/views/status_messages/_status_message.xml.builder

```diff
@@ -0,0 +1,7 @@
+builder.status_message(id: status_message.id) do |m|
+  m.message status_message.message
+  m.user status_message.user.login
+  m.severity status_message.severity
+  m.scope status_message.communication_scope
+  m.created_at status_message.created_at
+end
```

Comment:
Why do you change from attributes to elements?

Reply:
because element is the right way to do. The rule of thumb is that every instance method or attribute should be element with exception of id, which is a metadata.

Reply:
I would tend to agree, but you now changed the complete API. I can't upload the XML anymore I've always uploaded and get the same result. Can we try to avoid this?

Reply:
I'm not sure. First we changed it anyway adding new elements. Second XML or HTML, that's just views.

Reply:
and as you explained [here](https://github.com/openSUSE/open-build-service/pull/9590#discussion_r431958661), XML API isn't an UI, so because this endpoint isn't exposed by osc, we can change it something that make sense

Reply:
I'm not saying you can't do it or that it's impossible. I'm asking you to be a bit more backward compatible. That would only require us to change back to attributes right?

Reply:
> I'm not saying you can't do it or that it's impossible. I'm asking you to be a bit more backward compatible. That would only require us to change back to attributes right?

wrong. 

The XML view looks like:

```
<message severity="0">
a single sample message. contains some text ... attributes are optional and ignored when sent (put) to the api. the format of this example will be used as a convenient way to put a message.
</message>
```

After this PR it will look like:

```
<status message id=XXX>
  <message>blabla</message>
  <severity>information</severity>
  <scope>all_users/scope>
</status_message>
```

After your proposal it would look like:

```
<message severity="0" scope="0" id="XXX">blabla</message>
```

which IMO is wrong specially when you look into the 'index' route where you have

```
<status messages count='x'>
<message severity="0" scope="0">blabla</message>
<message severity="0" scope="0">blabla</message>  
</status_messages>
```

but the correct should be:

```
<status messages count='x'>
<status_message id="1" >...</status_message>
<status_message id="2">...</status_message>  
</status_messages>
```

p.s the id will be necessary in the destroy, which I didn't want to change now, but as all API destroy endpoints it should get the id from the XML and not from the URL

https://build.opensuse.org/apidocs/index#174

Reply:
Okay, did not know this was this untidy. Let's declare this a breaking change because we clean up the `StatusMessage` API :+1: 

Please add something to the release notes about this.

Reply:
@hennevogel Release notes updated